bigquery_etl/cli/monitoring.py

```diff
@@ -48,31 +49,58 @@ def monitoring(ctx):
 @sql_dir_option
 @click.option(
     "--workspace",
-    default="DEFAULT",
+    default=463,
     help="Bigeye workspace to use when authenticating to API.",
 )
+@click.option(
+    "--api-key",
+    "--api_key",
+    required=True,
```

Comment:
I wonder if this should be an option, it sounds like at some point we could potentially end up with plain text API key being passed into the command at some point and getting stored in some logs? Maybe we could just have a check inside the function to make sure the env variable is defined?

Reply:
I was thinking it would be slightly more convenient when running the tooling locally. This way users get an error automatically if the env variable is not set and at the same time see that an API key is required when calling `bqetl monitoring deploy --help`.

I'll remove the option for now just to be safe. I guess we can add it back later if it turns out to be more convenient